pep-0001.txt

```diff
@@ -520,7 +519,7 @@ PEP Formats and Templates
 =========================
 
 PEPs are UTF-8 encoded text files using the reStructuredText_ format.
-ReStructuredText_ allows for rich markup that is still quite easy to
+ReStructuredText allows for rich markup that is still quite easy to
```

Comment:
```suggestion
reStructuredText allows for rich markup that is still quite easy to
```
https://docutils.sourceforge.io/rst.html styles it as lowercase even at the beginning of a sentence.